app/uk/gov/hmrc/cgtpropertydisposals/models/upscan/UpscanUploadStatus.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cgtpropertydisposals.models.upscan
+
+import cats.Eq
+import julienrf.json.derived
+import play.api.libs.json.OFormat
+
+sealed trait UpscanUploadStatus
+
+object UpscanUploadStatus {
+  case object Initiate extends UpscanUploadStatus
```

Comment:
is it more accurate to have `Initiated` in the past tense here?

Reply:
done